concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Lastra-1986-381-1	1	agua	water	948	WATER
+Lastra-1986-381-2	2	fuego	fire	221	FIRE
+Lastra-1986-381-3	3	humo	smoke	778	SMOKE (EXHAUST)
+Lastra-1986-381-4	4	hollín	soot		
+Lastra-1986-381-5	5	ceniza	ash	646	ASH
+Lastra-1986-381-6	6	carbón	coal	1188	CHARCOAL
+Lastra-1986-381-7	7	tierra	land	1228	EARTH (SOIL)
+Lastra-1986-381-8	8	arena	sand	671	SAND
+Lastra-1986-381-9	9	cerro	hill	639	MOUNTAIN
+Lastra-1986-381-10	10	cerros	hills		
+Lastra-1986-381-11	11	río	river	666	RIVER
+Lastra-1986-381-12	12	lama (en el agua)	Lama (in water)		
+Lastra-1986-381-13	13	cielo	sky	1732	SKY
+Lastra-1986-381-14	14	luna (en caso de préstamo, preguntar "mes")	moon	1313	MOON
+Lastra-1986-381-15	15	sol	sun	1343	SUN
+Lastra-1986-381-16	16	estrella	star	1430	STAR
+Lastra-1986-381-17	17	luz	light	630	LIGHT (RADIATION)
+Lastra-1986-381-18	18	sombra (de un árbol)	shadow (of a tree)	1388	SHADE
```

Comment:
A general question about this list: Would this be a specification i.e. 'shadow (like from a tree)' or indeed a separate concept 'the shadow of a tree'? If the latter, I'd unmap.

Reply:
I believe it is the former. I've seen through the list many cases like this so I treated them with the same strategy.